### DISCUSSION TITLE
Cloud services option

### DISCUSSION THREAD

**Discussion by pzeepzee** [LeoKlaus/plappa#206] [Feature Requests]
Hi wondering if you will consider expanding your audiobook platform to include additional storage options freon the popular cloud services. Adding support for popular cloud services like iCloud and Google Drive would make your service more accessible to users who may not be as tech-savvy and all that. This would also broaden your potential customer base I'd imagine. Cheers on a great app.

**Comment by LeoKlaus** [comment]
Hi pzeepzee,

I have actually considered that but have decided against it.
While offloading the files themselves to a cloud service is a pretty simple task, obtaining and sharing metadata is much more complex. I much rather spend my limited time working on improving the experience for my existing users for now.
